confd/templates/base-storm-topology/topology.properties.tmpl

```diff
@@ -96,6 +96,7 @@ floodlight.alive.timeout = {{ getv "/kilda_floodlight_alive_timeout" }}
 floodlight.alive.interval = {{ getv "/kilda_floodlight_alive_interval" }}
 message.blacklist.timeout = {{ getv "/kilda_message_blacklist_timeout" }}
 floodlight.regions = {{ getv "/kilda_floodlight_regions" }}
+# floodlight.switch.mapping.remove.delay.seconds = 900
```

Comment:
why is this commented, not removed?

Reply:
Why it must be removed? This is a new option and its default value has been added into example config as a comment.

Reply:
why you commented it by default then?

Reply:
em... How some "user" can know that such an option exists? The only other way to know - is to read code a lot of code...